c7n/resources/datapipeline.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2016 Capital One Services, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Data Pipeline
+"""
+from c7n.manager import resources
+from c7n.query import QueryResourceManager
+
+
+@resources.register('datapipeline')
+class DataPipeline(QueryResourceManager):
+
+    class resource_type(object):
+        service = 'datapipeline'
+        type = 'dataPipeline'
+        id = name = 'Cheese'
+        date = 'CreatedTime'
```

Comment:
should be None afaics this resources doesn't have a create date reported by the api.